systems/money.go

```diff
@@ -0,0 +1,116 @@
+package systems
+
+import (
+	"fmt"
+	"image/color"
+	"sync/atomic"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+	"engo.io/engo/common"
+	"github.com/EngoEngine/TrafficDefense/systems/ui"
```

Comment:
I thought about `TrafficManager` or `TrafficDefense` (the latter due to the new "nature" of the game as discussed in gitter). I guess I changed it to `TrafficDefense` locally.